app/controllers/concerns/foreman/controller/config_templates.rb

```diff
@@ -13,8 +13,9 @@ def load_vars_from_config_template
   private
 
   def default_template_url(template, hostgroup)
-    url_for(:host => Setting[:unattended_url], :action => :template, :controller => '/unattended',
-            :id => template.name, :hostgroup => hostgroup.name)
+    path = url_for(:only_path => true, :action => :template, :controller => '/unattended',
+                   :id => template.name, :hostgroup => hostgroup.name)
+    Setting[:unattended_url] + path
```

Comment:
Sorry what do you mean?

The URL renders correctly:
` ks=http://katello-dev.bitbin.de/unattended/template/Katello%20Kickstart%20Default/With%20Puppet`